front/app/containers/Admin/projects/edit/participationContext.tsx

```diff
@@ -472,13 +484,17 @@ class ParticipationContext extends PureComponent<
   };
 
   validate() {
```

Comment:
Maybe it would be nice to extract this `validate` method into a function in a seperate file. That way it will also be easy to add some unit tests, which seems like a good thing to do for validation-related things